archive/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db              import models
+from django.core.validators import MinValueValidator, MaxValueValidator
+from django.utils           import timezone
+
+from user.models    import User
+from content.models import Content
+
+
+class Rating(models.Model):
+     user        = models.ForeignKey(User, on_delete = models.CASCADE, related_name='rated_movie')
+     content     = models.ForeignKey(Content, on_delete = models.CASCADE, related_name='rating_user')
+     rating      = models.FloatField(validators=[MinValueValidator(0.5), MaxValueValidator(5.0)])
+     watch_date  = models.DateField(null=True)
+     create_time = models.DateTimeField(default = timezone.now)
```

Comment:
```suggestion
     created_at = models.DateTimeField(default = timezone.now)
```
데이터베이스에서 자료 생성 시간을 기록하는 컬럼은 `created_at`이라는 단어를 주로 사용합니다 :)

Reply:
내용 수정 완료했습니다!